### PR TITLE
adds tesh

### PR DESCRIPTION
**Pull request by timothyteakettle**
## About The Pull Request
adds tesh

## Changelog
:cl:
add: teshari
/:cl:

### CODE REVIEW

**Comment by LunarFleet** [comment]
Praise the man, bringer of the chickens
Praise he

**Comment by timothyteakettle** [comment]
most of whats left is just what i've been doing in these commits, just moving teshari sprites over so they have stuff to wear, which is now about halfway done, woo

**Comment by timothyteakettle** [comment]
wow this is getting very close to being done

**Comment by timothyteakettle** [comment]
this compiles and everything needed is done, however i'm still going through it to fix things which should take a few more days at most, want to make sure it's in the best possible form

**Comment by timothyteakettle** [comment]
fixed the bugs i'm aware of, gonna work on porting teshari hair next

**Comment by timothyteakettle** [comment]
basically done now, just testing until its needed so it can then be merged

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -1679,33 +1679,114 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 						hair_color = sanitize_hexcolor(new_hair, 6)
 
 				if("hair_style")
+					var/list/snowflake_hair_list = list()
```
Comment (cadyn) [inline code/modules/client/preferences.dm:1682]: Marking this as an example but these problems are throughout the file. Please remove any unnecessary edits, and mark ALL edits that are kept.

File: code/modules/clothing/clothing_lr.dm [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/obj/item/clothing
+	///Lunar Station Edit
```
Comment (cadyn) [inline code/modules/clothing/clothing_lr.dm:2]: files marked as _lr don't need edit marks. Would prefer these removed from cleanliness

File: code/modules/clothing/outfits/teshari.dm [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// put teshari outfits here
```
Comment (cadyn) [inline code/modules/clothing/outfits/teshari.dm:1]: Rename this file to teshari_lr please.

File: code/modules/clothing/suits/teshari.dm [hunk 4]
```diff
@@ -0,0 +1,683 @@
+/obj/item/clothing/suit/teshari/cloak
```
Comment (cadyn) [inline code/modules/clothing/suits/teshari.dm:1]: Same issue, should be called teshari_lr.dm

File: code/modules/clothing/under/_under.dm [hunk 5]
```diff
@@ -154,7 +154,12 @@
 			if((flags_inv & HIDEACCESSORY) || (A.flags_inv & HIDEACCESSORY))
 				return TRUE
 
-			accessory_overlay = mutable_appearance('icons/mob/clothing/accessories.dmi', attached_accessory.icon_state)
+			var/accessory_file = 'icons/mob/clothing/accessories.dmi'
```
Comment (cadyn) [inline code/modules/clothing/under/_under.dm:157]: Please mark this edit as Lunar Edit

File: code/modules/clothing/under/teshari.dm [hunk 6]
```diff
@@ -0,0 +1,118 @@
+/obj/item/clothing/under/teshari
```
Comment (cadyn) [inline code/modules/clothing/under/teshari.dm:1]: Once more, file should be titled teshari_lr.dm

File: code/modules/mob/living/carbon/human/species.dm [hunk 7]
```diff
@@ -517,6 +517,15 @@ GLOBAL_LIST_EMPTY(roundstart_race_names)
 		for(var/obj/item/bodypart/B in C.bodyparts)
 			B.change_bodypart_status(BODYPART_HYBRID, FALSE, TRUE) // Makes all Bodyparts 'robotic'.
 
+	if(max_health)
```
Comment (cadyn) [inline code/modules/mob/living/carbon/human/species.dm:520]: Needs to be marked as Lunar edit

File: code/modules/mob/living/carbon/human/species_types/teshari.dm [hunk 8]
```diff
@@ -0,0 +1,29 @@
+/datum/species/teshari
```
Comment (cadyn) [inline code/modules/mob/living/carbon/human/species_types/teshari.dm:1]: teshari_lr.dm

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 9]
```diff
@@ -162,6 +162,9 @@ There are several things that need to be remembered:
 					variant_flag |= clip_flag
 					mask = T.alpha_mask_state
 
+			if(dna.species.id in U.species_alternative)
```
Comment (cadyn) [inline code/modules/mob/living/carbon/human/update_icons.dm:165]: Please go through this file and mark all necessary edits as lunar edits.

File: code/modules/mob/living/carbon/update_icons.dm [hunk 10]
```diff
@@ -109,8 +109,13 @@
 		inv.update_icon()
 
 	if(wear_neck)
```
Comment (cadyn) [inline code/modules/mob/living/carbon/update_icons.dm:111]: All edits here need to be marked as well.

File: code/modules/surgery/bodyparts/_bodyparts.dm [hunk 11]
```diff
@@ -780,7 +780,7 @@
 			if(burnstate)
 				. += image('icons/mob/dam_mob.dmi', "[dmg_overlay_type]_[body_zone]_0[burnstate]", -DAMAGE_LAYER, image_dir)
 
-		if(!isnull(body_markings) && is_organic_limb(FALSE))
+		if(!isnull(body_markings) && (is_organic_limb(FALSE) || is_iconless_prosthetic))
```
Comment (cadyn) [inline code/modules/surgery/bodyparts/_bodyparts.dm:783]: More unmarked edits

File: modular_citadel/code/modules/client/loadout/teshari_cloaks.dm [hunk 12]
```diff
@@ -0,0 +1,552 @@
+/datum/gear/xenowear/tesh/cloak
```
Comment (cadyn) [inline modular_citadel/code/modules/client/loadout/teshari_cloaks.dm:1]: teshari_cloaks_lr.dm please.

File: modular_citadel/code/modules/client/loadout/xenowear.dm [hunk 13]
```diff
@@ -0,0 +1,94 @@
+/datum/gear/xenowear
```
Comment (cadyn) [inline modular_citadel/code/modules/client/loadout/xenowear.dm:1]: xenowear_lr.dm

File: code/__DEFINES/lunar_defines.dm [hunk 14]
```diff
@@ -0,0 +1,9 @@
+//xenowear for the loadout tab
```
Comment (cadyn) [inline code/__DEFINES/lunar_defines.dm:1]: These defines shouldn't be lumped together under a signle lunar_defines.dm, I'd rather them be added to bottom of their proper respective files with a lunaredit tag to keep the defines organized.

**Review by cadyn — CHANGES_REQUESTED** [review]
Basic Maintaining changes that need to be made.

File: code/modules/client/preferences.dm (hunk 1 above)
Comment (timothyteakettle) [reply code/modules/client/preferences.dm:1682]: this isn't an unnecessary edit

**Comment by timothyteakettle** [comment]
made the required changes

**Review by cadyn — COMMENTED** [review]
We test merge for a bit.

**Comment by Lunarbotss13** [comment]
#### Test Merge Deployed

##### Server Instance
lunarstation

##### Revision
Origin: 95c3b1200bdf9de1a36f711275421c9ed8ebf31f
Pull Request: 4fa163ad5da47bf08e93448ad42001f53c18e46b
Server: 8f7fb2d8736f37b1446a34c976b0b4284907171c

**Comment by timothyteakettle** [comment]
yeah that's fine, if anything else needs changing or fixing just leave it here and i'll get to it

**Comment by cadyn** [comment]
> yeah that's fine, if anything else needs changing or fixing just leave it here and i'll get to it

We've already discovered that IPCs and synthlizards seem to have their sprites broken as a result of this PR 
image

**Comment by timothyteakettle** [comment]
that's quite strange, but thanks for notifiying me i'll look into it now!

**Comment by Lunarbotss13** [comment]
#### Test Merge Updated

##### Server Instance
lunarstation

##### Revision
Origin: 3a6c58234a271f5f42a8dc5dd3fcd3615f6927c1
Pull Request: 99f4f0448e9273fc48f8e2a3c6cff628525394c6
Server: 83465199c8744de63c1558a50cb3c70f82d0fe3e

**Comment by Lunarbotss13** [comment]
#### Test Merge Updated

##### Server Instance
lunarstation

##### Revision
Origin: 49850abaa0396f724e81380549b5c30ecb8d114f
Pull Request: 99f4f0448e9273fc48f8e2a3c6cff628525394c6
Server: b9ab19a7f601b342268aaaa08953c88cc556113d

**Comment by Lunarbotss13** [comment]
#### Test Merge Updated

##### Server Instance
lunarstation

##### Revision
Origin: b29f407544333fd68c1e35207fb0e2f4c64a20d3
Pull Request: 99f4f0448e9273fc48f8e2a3c6cff628525394c6
Server: 48de057898175f777bfc477d97d3e4fd83e83a30

**Comment by Lunarbotss13** [comment]
#### Test Merge Updated

##### Server Instance
lunarstation

##### Revision
Origin: a6506c5a43a0db7d2b299faf8907864eaabd5a02
Pull Request: 99f4f0448e9273fc48f8e2a3c6cff628525394c6
Server: bfcd2f7c9e635739a46718d6e7e248bf7a72a23a

**Comment by Lunarbotss13** [comment]
#### Test Merge Updated

##### Server Instance
lunarstation

##### Revision
Origin: 9b589bf73fe24ec350fe2cfa53514cbf30f6a3e2
Pull Request: 99f4f0448e9273fc48f8e2a3c6cff628525394c6
Server: 47cb1ade0303b1ffc42bef651bac280e05719b8c

**Comment by Lunarbotss13** [comment]
#### Test Merge Removed

**Pull request merged**